src/main/java/net/consensys/wittgenstein/core/NetworkLatency.java

```diff
@@ -81,6 +83,32 @@ public int getLatency(Node from, Node to, int delta) {
     }
   }
 
+  public static class NetworkLatencyByCity extends NetworkLatency {
+
+    private final Map<String, Map<String, Float>> latencyMatrix;
+
+    public NetworkLatencyByCity(CSVLatencyReader csvLatencyReader) {
+      this.latencyMatrix = csvLatencyReader.getLatencyMatrix();
+    }
+
+    public int getLatency(Node from, Node to, int delta) {
+
+      String cityFrom = from.cityName.get();
+      String cityTo = to.cityName.get();
+
```

Comment:
Shall we divide it by 2? The measurements are for round trip time but we care about one way, the same question for other latency models